src/Highlight.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import moment from "moment";
+
+class Highlight extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      highlighted: false
+    };
+  }
+
+  changeColor() {
+    this.setState(previousState => {
+      return {
+        highlighted: !previousState.highlighted
+      };
+    });
+  }
+
+  render() {
+    return (
+      <tr
+        className={this.state.highlighted ? "highlighted" : " "}
+        onClick={() => this.changeColor()}
+      >
+        <td>{this.props.customer.id}</td>
```

Comment:
There are some advantages (as you go forward) to creating these `td`s by mapping through a list of the keys.  However, your approach is the currently very readable.  

You could also temporarily assign `this.props.customer` to another variable, to reduce the amount of `noise`.
`const cust = this.props.customer` at the start of `render()`, before the `return` statement begins.  That would allow you to then write:
```
<td>{cust.title}</td>
<td>{cust.firstName}</td>
```
 etc